tasks/localizationtasks.js

```diff
@@ -149,12 +148,62 @@ gulp.task('ext:localization:xliff-to-ts', function () {
 		.pipe(gulp.dest(config.paths.project.root + '/src/constants/'));
 });
 
+// Generates a package.nls.json file from localizedPackage.json.enu.xlf
+gulp.task('ext:localization:generate-eng-package.nls', function () {
+	return gulp.src([config.paths.project.localization + '/xliff/enu/localizedPackage.json.enu.xlf'], { base: '.' })
+		.pipe(through.obj(function (file, enc, callback) {
+			// convert xliff into json document
+			let dict = convertXmlToDictionary(String(file.contents), false);
+
+			var contents = ['{'];
+			var regxForReplacingQuots = new RegExp('"', 'g');
+
+			// Get all keys and values from localizedPackage.json.enu.xlf
+			Object.keys(dict).forEach(key => {
+				let value = '';
+				if (contents.length >= 2) {
+					contents[contents.length - 1] += ',';
+				}
+				if (dict.hasOwnProperty(key)) {
+					value = dict[key]['source'];
+				}
+				if (value && value.indexOf('"') >= 0) {
+					value = value.replace(regxForReplacingQuots, '\'');
+				}
+				let instantiation = '"' + key + '":"' + value + '"';
+				contents.push(instantiation);
+			});
+
+			// end the function
```

Comment:
What function? This is just a dictionary (or object)

Not sure this needs a comment regardless